.changeset/nasty-moons-deliver.md

```diff
@@ -0,0 +1,6 @@
+---
+"@justeattakeaway/pie-notification": minor
+"@justeattakeaway/pie-storybook": minor
+---
+
+[Added] - support xsmall size for actions button
```

Comment:
Nit: Could we separate these into two seperate changesets?

While this description makes sense for `pie-notification`, that context of a pie-notification change won't be present in the storybook changelog.

Something like this should be enough


```
---
"@justeattakeaway/pie-storybook": minor
---

[Added] - Updated pie-notification stories to support xsmall size for actions button
```

Reply:
On a separate note, this could be avoided in future if we move the Storybook stories to the component package, which personally I think makes more sense.

It's a very small thing, but perhaps something we can discuss as a team.

Reply:
updated